pkg/controllers/user/helm/common.go

```diff
@@ -174,13 +190,21 @@ func (l *Lifecycle) generateTemplates(obj *v3.App) (string, string, *common.Helm
 		return "", "", nil, err
 	}
 	if err := cmd.Start(); err != nil {
-		return "", "", nil, errors.Wrapf(err, "start helm template --notes failed. %s", sbErr.String())
+		return "", "", nil, errors.Wrapf(err, "start helm template rendering notes failed. %s", sbErr.String())
 	}
 	if err := cmd.Wait(); err != nil {
-		return "", "", nil, errors.Wrapf(err, "wait helm template --notes failed. %s", sbErr.String())
+		return "", "", nil, errors.Wrapf(err, "wait helm template rendering notes failed. %s", sbErr.String())
 	}
 	template := sbOut.String()
 	notes := noteOut.String()
+	if common.IsHelm3(obj.Status.HelmVersion) {
+		splitNotes := strings.SplitAfter(notes, "NOTES:")
```

Comment:
Is it guaranteed nothing comes out after notes? Especially things that can leak keys

Reply:
as you said, based on the source it is the last thing coming out. future changes could make this an issue.